src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -140,9 +140,14 @@ BuildAggregatePlan(Query *masterQuery, Plan *subPlan)
 	havingQual = masterQuery->havingQual;
 
 	/* estimate aggregate execution costs */
-	MemSet(&aggregateCosts, 0, sizeof(AggClauseCosts));
+	memset(&aggregateCosts, 0, sizeof(AggClauseCosts));
+#if (PG_VERSION_NUM >= 90600)
+	get_agg_clause_costs(NULL, (Node *) aggregateTargetList, AGGSPLIT_SIMPLE,
+						 &aggregateCosts);
+	get_agg_clause_costs(NULL, (Node *) havingQual, AGGSPLIT_SIMPLE, &aggregateCosts);
+#else
 	count_agg_clauses(NULL, (Node *) aggregateTargetList, &aggregateCosts);
-	count_agg_clauses(NULL, havingQual, &aggregateCosts);
```

Comment:
Hm, if we run the count stuff in MasterAggregateExpression(), why do we need to do again here? Or, asking the other way around, why do we need to set it in the former, if we're doing it here?